gamebench_api_client/models/dataframes/time_series/abstract_time_series.py

```diff
@@ -23,10 +22,11 @@ def __init__(self, **request_parameters):
                 build and send a response.
         """
 
-        username_and_password = get_username_and_password()
         super().__init__()
-        self.authenticator = Authenticator(username_and_password)
-        self.mediator = TimeSeriesMediator(**request_parameters)
+        self.authenticator = Authenticator()
+        self.request_parameters = request_parameters
```

Comment:
This can be deleted as it's duplicated from the base class.

Reply:
Nice catch!

Reply:
It was a pretty good catch 👍 
Removed in dce6d57!